cvd_src/Win32/glwindow.cpp

```diff
@@ -1,85 +1,88 @@
 #include <cvd/glwindow.h>
 #include <exception>
 
+#include <gl/gl.h>
 #include <windows.h>
 #include <windowsx.h>
-#include <gl/gl.h>
```

Comment:
I recommend you revert this particular change, or at least find a way around it; you need to `#include <windows.h>` before the OpenGL headers.